### ISSUE TITLE
Unique Open Contracting ID per contracting process?

### ISSUE THREAD

**Issue by JachymHercher** [open-contracting/standard#728]
Just wondering: is there any mechanism in OCDS to ensure that a contracting process has only a single ocid? For example, I would expect that a contracting process would receive an ocid in Contracts Finder and another, totally different, ocid in Digiwhist. 

I'm assuming there isn't (because OCDS does not try to control who's publishing what) and it's rather a matter for the publishers to solve. That's probably fine in the short-mid term, but I thought it's still worth flagging.

**Comment by jpmckinney** [comment]
OCDS has strategies to avoid the same OCID being used to refer to multiple contracting processes (e.g. using [OCID prefixes](http://standard.open-contracting.org/latest/en/schema/identifiers/#globally-unique-identifiers) to scope each publisher's identifiers), but doesn't have strategies to avoid the same contracting process having multiple OCIDs (e.g. if two publishers disclose information about the same contracting process).

In short, an OCID ought to refer to a unique contracting process, but a contracting process might not have a unique OCID. A major challenge to ensuring the latter is that an OCID is synthetic (i.e. it's not original to the procurement system) so it would require coordination (either between publishers or with a central service) to avoid minting multiple identifiers for the same process.
